### PR TITLE
feat(support) add lab class schedules; fix #147

### LINKED ISSUE #147: Add lab schedules for OM1350, OM1360

### PR DESCRIPTION
**Pull request by marcolussetti**
This creates a tabbed view for the page allowing to look at the calendars for OM1360 & OM1350.

I know this is missing OM1325 but we aren't maintaining a calendar for it ATM.

### CODE REVIEW

**Comment by frogcrush** [comment]
You see, I disagree with adding this. First, it bloats up the page with three separate GCals being loaded when visiting the page. Second, I don't really how this relates to the support lab. Third, I don't see how those GCal files would be accurate enough to bother posting. I mean, the Discord bot scrapes TRU, but I wouldn't consider it accurate by any means (it doesn't account for holidays, other room bookings, etc). 

tldr; I vote to reject this PR.

**Comment by frogcrush** [comment]
I think if we DID want to add schedules, we should write our own little thing that pulls data from the JSON file that I source the bots' info from.
